homeassistant/components/sensor/miflora.py

```diff
@@ -31,9 +29,7 @@
 DEFAULT_NAME = 'Mi Flora'
 DEFAULT_RETRIES = 2
 DEFAULT_TIMEOUT = 10
-
-UPDATE_INTERVAL = 1200
-MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=UPDATE_INTERVAL)
```

Comment:
Conf_cache is used in the Mi flora library, and is used to cache the different senors in the same device.
https://github.com/home-assistant/home-assistant/pull/5348/files#diff-1fc0de9c510059523b4bb4091c1fd7abR61